README.rst

```diff
@@ -0,0 +1,153 @@
+=======================
+EOX core |build-status|
+=======================
+
+.. |build-status| image:: https://circleci.com/gh/eduNEXT/eox-core.svg?style=svg
+
+Eox-core (A.K.A. Edunext Open eXtensions) is a `django app plugin](https://github.com/edx/edx-platform/tree/master/openedx/core/djangoapps/plugins) for adding multiple APIS for the [edx-platform <https://github.com/edx/edx-platform>`_ for adding multiple APIS for the `edx-platform <https://github.com/edx/edx-platform>`_ to make changing the platform easier on commercial environments, including bulk creation of pre-activated users (e.g. skip sending an activation email) and enrollments.
```

Comment:
@morenol  This description is a bit weird, what do you think of:

Eox-core (A.K.A. Edunext Open extensions) is a plugin (plugins-link) for the edx-platform that adds multiple API endpoints in order to extend the functionality of the edx-platform and avoid changing the base code directly. These API endpoints includes bulk creation of pre-activated users (for example, skip sending an activation email) and enrollments.

Reply:
I just took that from the old README, I didn't add anything

Reply:
But I will add your changes